riotfile.py

```diff
@@ -101,9 +101,6 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         "_DD_CIVISIBILITY_USE_CI_CONTEXT_PROVIDER": "1",
         "DD_TESTING_RAISE": "1",
         "DD_REMOTE_CONFIGURATION_ENABLED": "false",
-        "DD_CIVISIBILITY_AGENTLESS_ENABLED": "1",
-        "DD_CIVISIBILITY_CODE_COVERAGE_ENABLED": "1",
-        "DD_CIVISIBILITY_ITR_ENABLED": "1",
```

Comment:
It looks like `DD_CIVISIBILITY_ITR_ENABLED` defaults to True so that was removed. 

But for `DD_CIVISIBILITY_AGENTLESS_ENABLED` and `DD_CIVISIBILITY_CODE_COVERAGE_ENABLED`, I just see these set to `1` in the `hatch.toml` file but not anywhere else. Are they unused?

Reply:
`DD_CIVISIBILITY_AGENTLESS_ENABLED` is being removed because we don't want it in the `riotfile.py` (and we want it to be falsy anyway).

`DD_CIVISIBILITY_CODE_COVERAGE_ENABLED` is actually no longer a valid env var at all.